plugins/amazonq/codewhisperer/jetbrains-community/tst/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererProjectCodeScanTest.kt

```diff
@@ -373,5 +374,9 @@ class CodeWhispererProjectCodeScanTest : CodeWhispererCodeScanTestBase(PythonCod
         // The tests include the markdown file but not these two files.
         projectRule.fixture.addFileToProject("/.gitignore", "node_modules\n.idea\n.vscode\n.DS_Store").virtualFile
         projectRule.fixture.addFileToProject("test.idea", "ref: refs/heads/main")
+
```

Comment:
please don't change other teams test unless necassary. Change the tests for our team instead. `FeatureDevSessionContextTest.kt::testZipProject`